packages/fluent/scss/datetimepicker/_layout.scss

```diff
@@ -10,7 +10,7 @@
     // Datetime selector
     .k-datetime-selector {
         display: flex;
-        transition: transform .2s;
+        @include kendo-motion(short-4, ease, transform);
```

Comment:
In the [motion easings list ](https://github.com/telerik/kendo-themes/pull/4779/commits/6af3b3b826d5c098967460b3d954abed183e5946#diff-2fbd99b1fa8fb0dd6af4da97661b0d7b5c4270893c91fa6ec813c8488d507e20R23)is missing `ease` - please change it with the correct one

Reply:
You are correct, I was able to work around that, but let me know what you think.